rust/foxglove/src/websocket/ws_protocol/server/server_info.rs

```diff
@@ -26,6 +26,12 @@ pub struct ServerInfo {
     /// Optional string.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub session_id: Option<String>,
+    /// Optional timestamp, in absolute nanoseconds, indicating the start of the data range.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub data_start_time: Option<u64>,
```

Comment:
u64 is problematic for parsing the JSON in javascript, because javascripts number type can't hold integers higher than 2^53 - 1 without loosing precision.

`{ sec: u32, nsec: u32 }` would work, but I wonder if we should not put the start and end time in the playback state struct?